app/services/document_capture_session_result.rb

```diff
@@ -10,16 +10,20 @@
   :failed_back_image_fingerprints,
   :captured_at,
   :selfie_check_performed,
-  :doc_auth_success, :selfie_success,
+  :doc_auth_success, :selfie_status, :selfie_success,
   keyword_init: true,
   allowed_members: [:id, :success, :attention_with_barcode, :failed_front_image_fingerprints,
                     :failed_back_image_fingerprints, :captured_at, :selfie_check_performed,
-                    :doc_auth_success, :selfie_success]
+                    :doc_auth_success, :selfie_status, :selfie_success]
 ) do
   def self.redis_key_prefix
     'dcs:result'
   end
 
+  def selfie_status
```

Comment:
similarly , result[:selfie_status] or result['selfie_status'] may have problem. But not critical.

Reply:
Are there any scenarios in which we read `session_result[:selfie_status]` or `session_result['selfie_status']`?  I thought we always used `session_result.selfie_status`.

Reply:
@eileen-nava , no just for completeness purpose.